src/NuGet.Core/NuGet.Configuration/Settings/Items/RepositoryItem.cs

```diff
@@ -199,8 +199,12 @@ internal override void Update(SettingItem other)
                 {
                     XElementUtility.RemoveIndented(_owners.Node);
                     _owners = null;
-                    Origin.IsDirty = true;
                     Owners.Clear();
+
```

Comment:
Does this need a test like the other ones?

Reply:
Other items do not have tests for this since it is only avoiding an unnecessary exception.